src/python/pants/backend/python/goals/setup_py_test.py

```diff
@@ -576,10 +587,23 @@ def assert_sources(
         expected_package_data,
         addrs,
     ):
-        targets = Targets(rule_runner.get_target(addr) for addr in addrs)
+        # We synthesize an owner for the addrs, so we have something to put in SetupPyChrootRequest.
+        rule_runner.create_file(
+            "src/python/foo/BUILD",
+            textwrap.dedent(
+                f"""
+        python_distribution(
```

Comment:
This is a bit funky, but if black likes it, I'm happy with it.

Reply:
That is a mistake after black breaks lines, it fails to indent that line properly. It accepts it just as well when indentend less funky.. it just won't touch it.. so you have to manually fix it.